### DISCUSSION TITLE
Reload CSS styling on the fly

### DISCUSSION THREAD

**Discussion by ghost** [phuhl/linux_notification_center#251] [Q&A] [answered]
The README says _Reload CSS styling on the fly_, I assume this means I can update the css style with like a command then reload it. But I didn't find any way to properly reload the css file.

Typically, I have pywal setup, and I'd like to take colors generated by pywal and apply them to the css styling, but I didn't find resources or discussion talking about this. 

If there is an intended way to do that, I would appreciate any advice :pray:

**Comment by phuhl** [comment, marked as answer]
Hey, you can reload the CSS file by running 

```
notify-send.py a --hint boolean:deadd-notification-center:true \
                 string:type:reloadStyle
```

Hope that helps.
